libraries/python2-buildtime/install.sh

```diff
@@ -9,3 +9,5 @@ apt-get install -u -y \
     python-pip
 
 pip install --upgrade pip
+
+rm -rf /var/lib/apt/lists/*
```

Comment:
nit: ideally, don't you want to delete the files that have a create timestamp greater than when we started running this script?

Reply:
do you mean just within the `/var/lib/apt` directory? it starts out as empty in the base ubuntu image (and most others i've tested with)

Reply:
Yeah, just talking about /var/lib/apt/lists/. I am guessing that you did this to keep the images small. And I wanted to make sure that we weren't adding layers for the removal of files from the base. If it's empty, this lgtm